### DISCUSSION TITLE
Is there a way to add headers to the response schema?

### DISCUSSION THREAD

**Discussion by iressstevenson** [apiflask/apiflask#496] [Q&A] [answered]
In the input we can specify location but not in the output.
Is there are way to specify output headers for the api spec?

**Comment by greyli** [comment]
I have created #507 to track this feature. Thanks!

**Reply by greyli** [reply, marked as answer]
Implemented in https://github.com/apiflask/apiflask/pull/511 and released with 2.1.0
